### DISCUSSION TITLE
Should we remove the in-memory message pump/secret provider?

### DISCUSSION THREAD

**Discussion by stijnmoreels** [arcus-azure/arcus.testing#236] [Q&A] [answered]
Not sure if this is something that is particulary useful for developers of regular applications. It might be better to allow people create their own test infrastructure.

Plus, since these are not commonly used fixtures (message pump + secret store), it might be unnecessary for now to support dedicated test infrastructure for them.

**Comment by fgheysels** [comment]
Don't base the decision on one single opinion, but I have never used or required the in-memory messagepump.

Typically we have unit-tests that test the message-handlers, but to test that, we don't need a messagepump.

**Comment by stijnmoreels** [comment, marked as answer]
Closed discussion as the direct link with Messaging is being removed.
